.vscode/settings.json

```diff
@@ -8,6 +8,7 @@
         "*.snap": "javascriptreact"
     },
     "eslint.enable": false,
+    "flow.enabled": false,
```

Comment:
this probably needs to go in most of our TS projects - I have a PR about this, but TBH I'm not sure if it's worth adding this at the extension level: https://github.com/flowtype/flow-for-vscode/pull/95